src/lib.rs

```diff
@@ -654,11 +654,13 @@ impl Builder {
         }
 
         if self.input_headers.len() > 1 {
-            output_vector.extend(
-                self.input_headers[..self.input_headers.len() - 1]
-                    .iter()
-                    .cloned(),
-            );
+            // To pass more than one header, we need to pass all but the last
+            // header via the `-include` clang arg
+            output_vector.reserve(2 * self.input_headers.len() - 2);
```

Comment:
I don't think this code is perf sensitive enough to warrant the `reserve()` call, fwiw.

Reply:
Yeah, probably not. I was imagining cases where someone called bindgen from their application (not just the build script) and figured it might be worth reducing the number of allocations.

Also, the call is safe and difficult to misuse. Even an off-by-one (or by two) "error" would not affect correctness.

Reply:
I removed it